lib/loaders/api/loader_one_off_factory.js

```diff
@@ -7,12 +7,19 @@ import { loaderInterface } from "./loader_interface"
  * This factory provides a short-cut system for our data loader system, it provides
  * and uncached result that is close to the equivilent to calling the fetch request directly.
```

Comment:
Reading this doesn’t really tell me what it does nor do I like the name for the same reason. Is it the same as an unauthenticated data loader, in that it does cache per query execution but _not_ to memcache? Or is it completely different in other ways? /cc @orta

(PS There’s some spelling errors here.)

Reply:
I built it to basically to conform to the same interfaces as the auth/unauth factories in user-code, but to skip any caching anywhere

Reply:
Maybe the data loaders shape changed since then? As I mentioned in the PR description, if we're adverse to the refactor we can just drop this commit: https://github.com/artsy/metaphysics/commit/3fa755d611711a21957dce4321ad8ed05e08ebe2